cve_bin_tool/cvedb.py

```diff
@@ -567,6 +619,12 @@ def populate_metrics(self):
         self.connection.commit()
         self.db_close()
 
+    def populate_epss(self):
```

Comment:
Another good place for a docstring. I feel like this short function is mostly self-evident to people who know the acronyms but could be difficult for someone who doesn't. So this is a nice opportunity to expand the acronym and have a really short explainer about what it's for.  Something like "Add Exploit Prediction Scoring System (EPSS) data to help users evaluate risks"